src/columns.jl

```diff
@@ -261,7 +275,10 @@ function sortperm(c::Columns)
     return p
 end
 
+sortperm(c::Columns{<:Pair}) = sortperm(c.columns.first)
```

Comment:
Where is this used? Just checking if this is OK.

Reply:
Hmm, I think we should probably remove this for now if it's not getting used anywhere. Maybe error messages will guide us to the right implementation.

Reply:
Well, but it allows `sort` and `sort!` to work out of the box with the `Columns{Pair}` (sorting by indexing `Columns`) and is in line with `issorted(c::Columns{<:Pair}) = issorted(c.columns.first)` which is quite useful to verify that our `keys => values` `Columns{Pair}` object is indeed sorted by `keys`.

Reply:
If you sort a vector of pairs (`sort([1=>2,1=>1])`) it sorts by both keys and values. Hence my reluctance. We should use different names for these if possible.

Reply:
Good point, I'll change `sort` related functions to do that then. The sorting for pairs would then simply be lexicographic, right?

Reply:
`sortperm(rows(c))` would do the trick if `rows(c)` strips the pairs... Or else you could use `refine_perm!` manually.